plugin/pkg/scheduler/api/types.go

```diff
@@ -139,10 +139,15 @@ type ExtenderArgs struct {
 	Nodes api.NodeList `json:"nodes"`
 }
 
+// FailedNodesMap represents the filtered out nodes map with node name and failed message
+type FailedNodesMap map[string]string
```

Comment:
Please don't put this in api/types.go

My suggestion is to just use raw type `map[string]string`